dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.7 as pyhome
```

Comment:
I'm going to leave this bit for now as I can see you haven't submitted it for M5, but when you are ready to submit, don't forget to update your Readme to clearly specify how a new user should build & run both your dev & prod images